### PR TITLE
fix(cookie): change default config

### PR DESCRIPTION
**Pull request by RomainLanz**
Hey there! 👋🏻 

This PR changes the default settings for cookies to ensure we are securing the cookie in production and using the correct default value for `sameSite`.

### CODE REVIEW

File: config/app.ts [hunk 1]
```diff
@@ -32,7 +33,7 @@ export const http = defineConfig({
     path: '/',
     maxAge: '2h',
     httpOnly: true,
-    secure: false,
-    sameSite: false,
+    secure: app.production,
```
Comment (thetutlage) [inline config/app.ts]: Is there a property called `app.production`? I think it is `inProduction`, but please double check

File: config/app.ts [hunk 2]
```diff
@@ -32,7 +33,7 @@ export const http = defineConfig({
...
+    sameSite: 'lax',
```
Comment (thetutlage) [inline config/app.ts:37]: Last time we had some issues with this when using Ally for social auth. Ally creates a CSRF cookie (for added security) and post redirect we were not able to read that cookie.

So have to double check what exactly the issue was and if we need to some notes in the docs for the same

**Review by thetutlage — CHANGES_REQUESTED** [review]

File: config/app.ts (hunk 2 above)
Comment (RomainLanz) [reply config/app.ts:37]: The `SameSite` attribute expects to have one of those 3 values: `Strict`, `LAX`, or `None`. That will define when the cookie is sent.

[Looking out the underlying library](https://www.npmjs.com/package/cookie#samesite) we are using (`cookie`), using the value `false` means the `SameSite` attribute will not be set.

If the `SameSite` attribute is not set, it is defaulted to a value defined by the browser. [Chrome moved to `LAX` a while ago](https://web.dev/samesite-cookies-explained/#samesite=lax-by-default).

Comment (thetutlage) [reply config/app.ts:37]: I get the technicalities behind it. But, can we test if this change impacts the Ally authentication behavior?

Comment (RomainLanz) [reply config/app.ts:37]: I have no issue with `Ally` using those values.
Also, as said, it is the default value set in many browsers when we use `none`.

**Pull request merged**